_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction
+Wasting a lot of time on building a project is a programming problem. At the moment a programmer starts an
+assembly, he loses focus on a task and spends valuable working time. Different build systems use many tools,
+helping to assemble a project faster, namely caching, task parallelization, distributed building and much more.
+The subject of this article is caching, because completed tasks caching allows not to spend resources again. 
```

Comment:
@Yanich96 

"The subject of this article is caching."

The other is obvious: 
> because completed tasks caching allows not to spend resources again